cmd/lotus-fountain/site/index.html

```diff
@@ -1,7 +1,9 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-    <title>Lotus Fountain</title>
+    <title>Filecoin Lotus Faucet – Get Free tFIL & DataCap</title>
+    <meta name="description" content="Get free tFIL and DataCap on the Filecoin Lotus Fountain Faucet by ChainSafe. Quickly connect your wallet, request test tokens, and start building or experimenting on the Filecoin testnet with ease.">
```

Comment:
```suggestion
    <meta name="description" content="Get free tFIL and DataCap in the Lotus Fountain. Quickly connect your wallet, request test tokens, and start building or experimenting on the Filecoin testnet with ease.">
```

ChainSafe Infra operates its own fork of the Lotus Fountain, which may have diverged from our current codebase. Additionally, the Lotus Fountain is deployed on an ad-hoc basis when the Butterfly Network is active with Lotus Infra, managed by FilOz. Therefore, I think it is best to avoid including specific names in the code.